### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  java.lang.ClassCastException: com.intellij.execution.junit.JUnitConfiguration cannot be cast to com.microsoft.azure.hdinsight.spark.run.configuration.LivySparkBatchJobRunConfiguration

### ISSUE THREAD

**Issue by alejandromachado** [Microsoft/azure-tools-for-java#3016]
IntelliJ build version: 2018.3.1 IC-183.4588.61
OS: Mac OS X
JDK: JetBrains s.r.o 1.8.0_152-release
Plugin version: 3.18.0-2018.3
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,0,0,1440x900,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=java.lang.ClassCastException: com.intellij.execution.junit.JUnitConfiguration cannot be cast to com.microsoft.azure.hdinsight.spark.run.configuration.LivySparkBatchJobRunConfiguration
	at com.microsoft.azure.hdinsight.spark.actions.SparkSubmitJobAction.submit(SparkSubmitJobAction.kt:85)
	at com.microsoft.azure.hdinsight.spark.actions.SparkSubmitJobAction.onActionPerformed(SparkSubmitJobAction.kt:64)
	at com.microsoft.azuretools.ijidea.utility.AzureAnAction.actionPerformed(AzureAnAction.java:57)
	at com.intellij.openapi.actionSystem.ex.ActionUtil$1.run(ActionUtil.java:258)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:275)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem$ActionTransmitter.lambda$actionPerformed$0(ActionMenuItem.java:287)
	at c

**Comment by wezhang** [comment]
Thanks @alejandromachado ! It has been fixed in v3.20

**Comment by wezhang** [comment]
Dup #2749
